lib/Dialect/BaseModelica/Transforms/EquationExpressionOpInterfaceImpl.cpp

```diff
@@ -3,13 +3,13 @@
 
 using namespace ::mlir::bmodelica;
 
-static void
-printExpression(llvm::raw_ostream &os, mlir::Value value,
-                const llvm::DenseMap<mlir::Value, int64_t> &inductions) {
+namespace {
+void printExpression(llvm::raw_ostream &os, const mlir::Value value,
```

Comment:
Remove `const` from basically everywhere you put it.
https://mlir.llvm.org/docs/Rationale/UsageOfConst/